front/lib/api/files/utils.ts

```diff
@@ -63,7 +63,8 @@ export const parseUploadRequest = async (
         return new Err({
           name: "dust_error",
           code: "file_too_large",
-          message: "File is too large.",
+          message:
+            "File is too large or the size passed to the File instance in the DB does not match the size of the uploaded file.",
```

Comment:
Hmm no need to mention "DB" here?

Reply:
It's the one in the DB i'm referring to, how would you call it ?

Reply:
@Fraggle "File instance in the DB" I think makes little sense to a developer (assuming this is developer visible).

Maybe File is too large or there is a mismatch of size between the data received and the size specified at creation?

The real question is why do we even need that field?

Reply:
And what happens when you set it above?